apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java

```diff
@@ -59,13 +59,25 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
-
-import static org.slf4j.LoggerFactory.getLogger;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.spi.CDI;
+import javax.inject.Singleton;
 
 public final class Shuffling {
+    /**
+     * Cache, which contains all active shufflings required for processing on each block push
+     * Purpose: getActiveShufflings db call require at least 50-100ms, with cache we can shorten time to 10-20ms
+     */
+    private static final Map<Long, Shuffling> activeShufflings = new HashMap<>();
```

Comment:
I propose name it as 'activeShufflingsCache' implicitly.

Reply:
Fixed